Source/Core/Core/HW/Memmap.cpp

```diff
@@ -307,6 +308,7 @@ u8* GetPointer(const u32 address)
 	default:
 		if (bFakeVMEM)
 			return m_pFakeVMEM + (address & FAKEVMEM_MASK);
+		break;
```

Comment:
This one mainly just silences a warning on higher warning levels, no functional change.